BoostTestUi/MainFrm.cpp

```diff
@@ -864,7 +864,7 @@ void CMainFrame::test_waiting(const std::wstring& processName, unsigned processI
 {
 	EnQueue([this, processName, processId]()
 	{
-		if (this->MessageBox(
+		if (m_devEnv.AttachDebugger(processId) || this->MessageBox(

```

Comment:
@djeedjay 
This is where the "auto-attach" is integrated: right before the old "Attach debugger ..." dialog is shown.
It still is shown if the auto-attach was unsuccessful.